packages/vx-brush/src/BaseBrush.tsx

```diff
@@ -280,16 +297,16 @@ export default class BaseBrush extends React.Component<BaseBrushProps, BaseBrush
     });
   };
 
-  reset() {
+  reset = () => {
     const { width, height } = this.props;
-    this.update(() => ({
-      start: undefined,
-      end: undefined,
+    this.updateBrush(() => ({
+      start: { x: 0, y: 0 },
+      end: { x: 0, y: 0 },
       extent: {
-        x0: undefined,
-        x1: undefined,
-        y0: undefined,
-        y1: undefined,
+        x0: 0,
```

Comment:
these are the values the brush is initialized with, we might reconsider this to be `-1` in the future. non-numbers would have to be handled in a lot of places.